mne_bids/write.py

```diff
@@ -492,9 +492,13 @@ def _sidecar_json(raw, task, manufacturer, fname, kind, overwrite=False,
     powerlinefrequency = raw.info.get('line_freq', None)
     if powerlinefrequency is None:
         powerlinefrequency = _estimate_line_freq(raw)
-        warn('No line frequency found, defaulting to {} Hz '
-             'estimated from multi-taper FFT '
-             'on 10 seconds of data.'.format(powerlinefrequency))
+        if powerlinefrequency is None:
+            powerlinefrequency = -1  # must be int for validator
+            warn('No line frequency found')
```

Comment:
you don't test this

Reply:
But CodeCov says I have 100% coverage ;)

Fixing it now